app/javascript/components/signup_form.jsx

```diff
@@ -0,0 +1,74 @@
+// eslint-disable-next-line no-use-before-define
+import React from "react";
+import $ from "jquery";
+
+function SignupForm () {
+  const [name, setName] = React.useState("");
+  const [sname, setSname] = React.useState("");
+  const [email, setEmail] = React.useState("");
+  const [password, setPassword] = React.useState("");
+  const [confirmpassword, setConfirm] = React.useState("");
+  const [errorsList, setErrors] = React.useState([]);
+
+  const SendForm = (e) => {
+    e.preventDefault();
+    $.ajax({
+      async: false,
+      type: "POST",
+      url: "/create",
+      data: {
+        first_name: name,
+        second_name: sname,
+        email: email,
+        password: password,
+        password_confirmation: confirmpassword
+      },
+
+      success: function (result) {
+        setErrors(result.errors);
+      },
+      error: function (result) {
+        alert("Please try again");
+      }
+    });
+  };
+  const listErrors =
+        errorsList.map((i, errorsList) =>
+                <li key = {errorsList}>{i}</li>
```

Comment:
What does `i` represent here? Your key is set to `errorsList` and you are displaying `i`. To me, it's not very clear. Maybe you could improve the variable names so it's easier to understand what each value is.

Reply:
My bad, I meant to change this after I got it working, updated now to be clearer